pkg/slo-controller/nodemetric/node_event_handler.go

```diff
@@ -47,11 +45,7 @@ func (n *EnqueueRequestForNode) Create(e event.CreateEvent, q workqueue.RateLimi
 }
 
 func (n *EnqueueRequestForNode) Update(e event.UpdateEvent, q workqueue.RateLimitingInterface) {
-	newNode, oldNode := e.ObjectNew.(*corev1.Node), e.ObjectOld.(*corev1.Node)
-	// TODO, only use for noderesource
-	if !n.isNodeAllocatableUpdated(newNode, oldNode) {
```

Comment:
is there any reason for removing this check?

Reply:
Expect a more aggressive trigger to update NodeMetricSpec

Reply:
As the watch for ConfigMap is added, could you please list some other cases where `nodeMetric.spec` should get updated but `node.allocatable` is unchanged?

Reply:
Checking if Node.Allocatable changes is just an early performance optimization. It should be more aggressive when a node changes whatever happens.